test/parallel/test-require-symlink.js

```diff
@@ -22,8 +24,8 @@ if (common.isWindows) {
   // On Windows, creating symlinks requires admin privileges.
   // We'll only try to run symlink test if we have enough privileges.
   exec('whoami /priv', function(err, o) {
-    if (err || o.indexOf('SeCreateSymbolicLinkPrivilege') == -1) {
-      console.log('Skipped: insufficient privileges');
+    if (err || !o.includes('SeCreateSymbolicLinkPrivilege')) {
+      common.skip('Skipped: insufficient privileges');
```

Comment:
Might actually be worthwhile to lint for `Skipped: ` in strings in tests come to think of it.